01.HTML+CSS/src/styles/site.scss

```diff
@@ -0,0 +1,88 @@
+* {
+    margin: 0;
+    font-family: 'Open Sans';
+    box-sizing: border-box;
+}
+
+$dark-pink: #ab0534;
```

Comment:
it is more preferable to keep variables in a separate file, ex. "variables.scss"
cosidering, you have both font weight and colors as variables, it is also makes sense to keep them in different files